Makefile

```diff
@@ -76,3 +79,41 @@ manpage:
 
 build-container:
 	podman build -t ${QPC_VAR_PROGRAM_NAME} .
+
+lock-requirements: lock-main-requirements
+	rm -f requirements-build.txt requirements-build.in
+	$(MAKE) search-build-requirements
+	$(MAKE) lock-build-requirements
+	# run another rounds of search/lock for build requirements
+	# (build dependencies also have build dependencies after all :)
+	$(MAKE) search-build-requirements
+	$(MAKE) lock-build-requirements
+	$(MAKE) search-build-requirements
+	$(MAKE) lock-build-requirements
```

Comment:
Why is this repeated exactly twice like this? This is what the quipucords make target does, but I do not understand why it repeats.

Reply:
this search should be recursive, but I had no idea how to do it on a makefile. I added these as a guess on what seems to be a "good enough" rounds.

IMO the recursion should be something done on pybuild-deps side (integrate with pip-tools and recursively look for build dependencies + lock until no new dependencies are found)